modules/cert-manager-override-arguments.adoc

```diff
@@ -47,12 +47,17 @@ spec:
     overrideArgs:
       - '--v=2' <4>
 ----
-<1> Provide a comma-separated list of `<host>:<port>` nameservers to query for the DNS-01 self check. For example, `--dns01-recursive-nameservers=1.1.1.1:53`.
+<1> Provide a comma-separated list of nameservers o query for the DNS-01 self check. The nameservers can be specified either as `<host>:<port>`, for example, `--dns01-recursive-nameservers=1.1.1.1:53`, or use DNS over HTTPS (DoH), for example, `https://1.1.1.1/dns-query`. +
 <2> Specify to only use recursive nameservers instead of checking the authoritative nameservers associated with that domain.
 <3> Provide a comma-separated list of `<host>:<port>` nameservers to query for the Automated Certificate Management Environment (ACME) HTTP01 self check. For example, `--acme-http01-solver-nameservers=1.1.1.1:53`.
 <4> Specify to set the log level verbosity to determine the verbosity of log messages.
 <5> Specify the host and port for the metrics endpoint. The default value is `--metrics-listen-address=0.0.0.0:9402`.
 <6> You must use the `--issuer-ambient-credentials` argument when configuring an ACME Issuer to solve DNS-01 challenges by using ambient credentials.
++
+[NOTE]
+====
+DNS over HTTPS (DoH) is available starting from {cert-manager-operator} version 1.13 and above. For earlier versions, you can utilize plaintext DNS.
+====
```

Comment:
I don't think `plaintext DNS` is the correct phrase to refer to standard/traditional DNS server which is non-HTTPS.
How about `DNS Name Servers` instead?

Also, for removing ambiguity we should convey to users that in new version we continue support traditional DNS servers.

Reply:
1. How about `DNS nameservers` like other references in callouts? I will keep it consistent based on what is correct.
2. Since callout specifies that both the formats are valid, may be I can remove the `For earlier versions,....` sentence here which could avoid the confusion? Which would mean for earlier versions they should be using <host>:<port> only.

Reply:
looks good in current form now!